### ISSUE TITLE
Folder icons missing

### ISSUE THREAD

**Issue by singhemant** [vscode-icons/vscode-icons#2823]
Folders icons are not showing in my vscode, I have installed the extension as per instructions. The file icons show up but not the folder icons, any suggestions, where I can debug this? 

Thanks!
![image](https://user-images.githubusercontent.com/45749122/129230826-78534db3-c2aa-4bf0-af46-52c2a20ebdbe.png)

**Comment by robertohuertasm** [comment]
That's strange... Let's try using F1 > Icons: Restore Default Icon Manifest. Let me know if that works.

**Comment by singhemant** [comment]
Yup, that worked. Thanks so much, really appreciated. That was quickest fix ever!
![image](https://user-images.githubusercontent.com/45749122/129232110-f9e8fe64-949b-492d-bcf2-dc77c0efc114.png)

**Comment by robertohuertasm** [comment]
Glad it worked! 🎉
